tsconfig.json

```
@@ -8,7 +8,8 @@
       "$components/*": ["app/scripts/components/*"],
       "$styles/*": ["app/scripts/styles/*"],
       "$utils/*": ["app/scripts/utils/*"],
-      "$context/*": ["app/scripts/context/*"]
+      "$context/*": ["app/scripts/context/*"],
+      "$data-layer/*": ["data-layer/*"]
```

To keep the repo clean, could we move this into from the repository root to a subdirectory like src?

```suggestion
      "$data-layer/*": ["src/data-layer/*"]
```

---

☝️ Can we put `data-layer` in `app/scripts`? That is where all the 'scripts' are right now.

---

@oliverroick @hanbyul-here should I move it under `app/scripts/*` or create a new directory to do `src/data-layer/*`? I will say I personally dont understand why we have a dir under app named `/scripts` and why it was named that but maybe we can revisit naming later.. lmk which you both prefer here

---

If we want to keep it all in the same place it should go to `app/scripts`. But I agree, we should revisit that at some point, this `app/scripts` convention we're using in some projects is odd.

---

Updated, can I get a re-review?